hack/clear-route-status.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+clear_status ()
+{
+	MY_JSON_BLOB=$(curl -s -X GET http://localhost:8001/oapi/v1/namespaces/$1/routes/$2/)
+	if [ $(echo "$MY_JSON_BLOB" |jq '.kind') == '"Route"' ]; then
+		MODIFIED_JSON=$(echo "$MY_JSON_BLOB" | jq 'del(.status.ingress)')
+		curl -s -X PUT http://localhost:8001/oapi/v1/namespaces/$1/routes/$2/status --data-binary "$MODIFIED_JSON" -H "Content-Type: application/json" > /dev/null
+		echo "route status for route $2 in namespace $1 cleared"
+	else
+		echo "Route $2 not found in namespace $1"
+	fi
+}
+
+USAGE="Usage:
+./clear-route-status.sh [namespace] [route-name | ALL]
+
+To clear the status of all routes in all namespaces:
+oc get namespaces | awk '{if (NR!=1) print \$1}' | xargs -n 1 -I %% ./clear-route-status.sh %% ALL
+
+To clear the status of all routes in namespace default:
+./clear-route-status.sh defalt ALL
+
+To clear the status of route example in namespace default:
+./clear-route-status.sh default example
+
+NOTE: if a router that admits a route is running it will immediatly update the cleared route status 
+"
+
+
+if [ "$#" -ne 2 ] || [[ "$@" == *" help "* ]]; then
+	printf "$USAGE"
+	exit
+fi
+
+oc proxy > /dev/null &
+PROXY_PID=$!
+
+
+
+if [ "$2" == "ALL" ]; then
+	ROUTES=$(oc get routes --namespace=$1 2> /dev/null | awk '{if (NR!=1) print $1}')
```

Comment:
You can use a template instead of the pipe to awk to get just the names of the routes. 
Example: `oc get route -n default --template="{{range .items}} {{.metadata.name}} {{end}}"`